recipes/genesis2/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "genesis2" %}
+{% set version = "2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slaclab/Genesis-1.3-Version2/archive/v{{ version }}.tar.gz
+  sha256: 0c8be1fd80cfa8339e3e40b5e798ec596d339e9b743c46a187321a71b9766e12
+  patches:
+    - size.patch  # [win]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}              # [not win]
+    - {{ compiler('m2w64_fortran') }}        # [win]
```

Would suggest using MSYS2's C compiler as well

```suggestion
    - {{ compiler('c') }}                    # [not win]
    - {{ compiler('m2w64_c') }}              # [win]
    - {{ compiler('fortran') }}              # [not win]
    - {{ compiler('m2w64_fortran') }}        # [win]
```

---

Sounds like a good idea! I noticed this pattern at some other recipes as well. I will add your patch.